src/Modal.js

```diff
@@ -340,7 +338,7 @@ class Modal extends React.Component {
             onExiting,
             manager,
             ref: this.setModalRef,
-            style: { ...style, ...this.state.style },
+            style: { ...style, display: 'block', ...this.state.style },
```

Comment:
i think it'd be best to do this only when `animation` is disabled

Reply:
OK, I'll update.